README.md

```diff
@@ -43,6 +43,10 @@ Line Interface](http://aws.amazon.com/cli/) product detail page.
 - [Contributing to the CLI](#contributing-to-the-cli)
 - [License](#license)
 
+#### Security disclosures
+
+If you think you’ve found a potential security issue, please do not post it in the Issues.  Instead, please follow the instructions [here](https://aws.amazon.com/security/vulnerability-reporting/) or [email AWS security directly](mailto:aws-security@amazon.com).
```

Comment:
Nit: I personally don't like using `mailto:` and would probably prefer just putting the email address like we do on our [Code of Conduct](https://github.com/aws/amazon-ecs-cli/blob/master/CODE_OF_CONDUCT.md), but not a blocker.